### DISCUSSION TITLE
How to control the opening and hiding of the drop-down box?

### DISCUSSION THREAD

**Discussion by yxd94772** [dowjones/react-dropdown-tree-select#586] [Q&A]
I want to control it through other buttons

**Comment by gandhis1** [comment]
Use `inlineSearchInput` and then you can control rendering of entire component or selectively using CSS.

**Reply by yxd94772** [reply]
> 使用`inlineSearchInput`然后您可以控制整个组件的呈现或选择性地使用 CSS。

thank you for your reply , Using inlineSearchInput just moves the search to the drop-down list, I want to use an extra button to control whether or not to showDropdown , Just using CSS doesn't seem right, maybe I don't understand 。

**Reply by yxd94772** [reply]
> 
I solved this problem by using showDropdown={'always'} to make the dropdown hold and then overwriting it with a style display. I don't know if this component has a way of controlling dropdown toggles
